### PR TITLE
`convert-pr-to-draft-improvements` - Improve style

### PR DESCRIPTION
**Pull request by kovsu**
## Test URLs

https://github.com/refined-github/sandbox/pull/98

## Screenshot

Before: image 

After: image

### CODE REVIEW

**Comment by kovsu** [comment]
image

**Comment by kovsu** [comment]
I don't know why the dialog doesn't close after I click.

image

**Comment by kovsu** [comment]
Don't want fix the new view issue here 🙁

File: source/features/convert-pr-to-draft-improvements.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function closeModal({delegateTarget: button}: DelegateEvent<MouseEvent, HTMLButt
 }
 
 function addConvertToDraftButton(alternativeActions: Element): void {
+	alternativeActions.classList.add('d-inline-flex', 'gap-1');
```
Comment (fregante) [inline source/features/convert-pr-to-draft-improvements.tsx:17]: Same as https://github.com/refined-github/refined-github/pull/8159#issuecomment-2550558616

In this case it seems you only need to call `convertToDraft.classList.replace('d-inline', 'd-inline-block')`

Comment (kovsu) [reply source/features/convert-pr-to-draft-improvements.tsx:17]: OOOOOOOOOO

**Comment by jabari-max** [comment]
> ## Test URLs
> 
> 
> 
> https://github.com/refined-github/sandbox/pull/98
> 
> 
> 
> ## Screenshot
> 
> 
> 
> Before: image 
> 
> 
> 
> After: image
> 
>

**Pull request merged**